packages/components/text-field/src/text-field.js

```diff
@@ -0,0 +1,163 @@
+import '@tradeshift/elements.input';
+import '@tradeshift/elements.help-text';
+import { customElementDefineHelper, html, TSElement, unsafeCSS } from '@tradeshift/elements';
+import css from './text-field.css';
+
+export class TSTextField extends TSElement {
+	constructor() {
+		super();
+		this.type = 'text';
+		this.id = 'input-id';
+		this.label = '';
+		this.value = '';
+		this.placeholder = '';
+	}
+
+	static get styles() {
+		return [TSElement.styles, unsafeCSS(css)];
+	}
+
+	static get properties() {
+		return {
+			/** Label of the text field. If you need something more than simple string, use the label slot. */
+			label: { type: String, reflect: true },
+			/** Id of the text field  */
+			id: { type: String, reflect: true },
+			/** Value of the text field */
+			value: { type: String, reflect: true },
+			/** Pass type of the input element, if it's not multiline */
+			type: { type: String, reflect: true },
+			/** Placeholder of the text field */
+			placeholder: { type: String, reflect: true },
+			/** Array of messages to pass to helpt-text component. See help-text component for more info  */
+			helpTextMessages: { type: Array, reflect: true, attribute: 'help-text-messages' },
+			/** If you have more than one help text message , you should pass a title to it. See help-text component for more info  */
+			helpTextTitle: { type: String, reflect: true, attribute: 'help-text-title' },
+			/** Error messages to show underneath of the input when it has error */
+			errorMessages: { type: Array, reflect: true, attribute: 'error-messages' },
+			/** Error title, if there are more than one error message */
+			errorTitle: { type: Array, reflect: true, attribute: 'error-title' },
```

Comment:
I'm pretty sure it's type should be a String instead of Array :)

Reply:
You are absolutely right :) Wondering how it did work in the storybook screenshot 🤔

Reply:
In Happo stories there is only one error message, so `error-title` is not used there.

Reply:
I meant the PR description screenshot: 
https://github.com/Tradeshift/elements/pull/333#issue-630368033